blueoil/cmd/train.py

```diff
@@ -395,8 +395,8 @@ def train(config_file, experiment_id=None, recreate=False):
     if not tf.io.gfile.exists(checkpoint):
         raise Exception('Checkpoints are not created in {}'.format(experiment_dir))
 
-    with open(checkpoint) as stream:
-        data = yaml.load(stream)
+    with tf.io.gfile.GFile(checkpoint) as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
```

Comment:
is this loader change related to GFile change?

It seems there are a few other use of `yaml.load` within blueoil thus maybe we should fix all in another diff.

Reply:
no, this is not directly related.

I just noticed that `load` method has been changed to specify `Loader` . 
Let me dedicate them as you commented.
https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

Reply:
fixed